apps/site/src/style.css

```
@@ -1,39 +1,59 @@
   --r-sm: var(--radius-sm);
-  --r-md: var(--radius-sm);
-  --r-lg: var(--radius);
-  --r-xxl: var(--radius);
+  --r-md: var(--radius-md);
+  --r-lg: var(--radius-lg);
   --r-pill: var(--radius-pill);
+  --shadow-card: var(--shadow-md);
+
   color-scheme: light;
-  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', helvetica, arial, sans-serif;
+  font-family: 'Geist', 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)